sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -41,3 +43,13 @@ export async function createModelClient(
   const credential = new AzureKeyCredential(apikey);
   return createClient(endpoint, credential, recorder?.configureClientOptions(options ?? {}));
 }
+
+export async function createEmbeddingsClient(
+  recorder?: Recorder,
+  options?: ClientOptions,
+): Promise<ModelClient> {
+  const endpoint = assertEnvironmentVariable("AZURE_EMBEDDINGS_ENDPOINT");
+  const apikey = assertEnvironmentVariable("AZURE_EMBEDDINGS_CLIENT_SECRET");
+  const credential = new AzureKeyCredential(apikey);
```

Comment:
Is token credential still not working?

Reply:
There's an issue with Vitest browser tests and importing @azure-tools/test-credential where the browser build fails. [This issue](https://github.com/Azure/azure-sdk-for-js/issues/30297) mentions it, but I think their investigation has shifted to working around by not using the vitest/test-cred combo. I will try again using a newer test-credential version once this SDK's v2 is released.

DefaultAzureCredential works in normal usage for me, and in Node Vitest tests, just not in browser Vitest tests.

Reply:
`createTestCredential` from the recorder package should work in both NodeJS and the browser. For the browser tests to build successfully, you probably need tsconfig similar to this [one](https://github.com/Azure/azure-sdk-for-js/blob/eh-tshy/sdk/eventhub/event-hubs/tsconfig.browser.config.json).

Reply:
I believe the current tsconfig.browser.config.json matches that exactly.